pydantic_ai_slim/pydantic_ai/models/function.py

```diff
@@ -277,4 +285,6 @@ def _estimate_usage(messages: Iterable[ModelMessage]) -> result.Usage:
 
 
 def _estimate_string_usage(content: str) -> int:
+    if not content:
+        return 0
```

Comment:
I think empty strings should count as zero tokens; the current logic returns `1` for an empty string. I believe this may be why some of the usage counts have gone down by one (or a small number anyway).